.gitignore

```diff
@@ -210,8 +210,8 @@ gradle-app.setting
 ### Docker ###
 
 # Ignore docker-compose
-**/docker-compose*.yaml
-**/docker-compose.yaml
+#**/docker-compose*.yaml
```

Comment:
why not remove them ?

Reply:
hi, when i removed them it try to delete the online version. i need to comment it here to assure it will stay in repo and not be deleted

this was a file which was not tracked with local changes but this somehow get annulated with the latest change which fabian did to support release on master. will try to get it back to be untracked after this commit

i was to try following after committ:
- Update .gitignore to ignore all future docker-compose.yaml files, except the one already in the repo.
- Use git update-index --assume-unchanged docker/docker-compose.yaml to stop tracking future local changes to that file.
- Push the updated .gitignore

in a case you have some other suggestion please ping me on the chat.
Thanks